packages/database/src/schemas/file.ts

```diff
@@ -146,6 +146,8 @@ export const files = pgTable(
       onDelete: 'set null',
     }),
 
+    slug: text('slug').unique(),
```

Comment:
1. Use `varchar255`
2. unique will cause the slug of documents for all users to be non-duplicable. I think this scope is a bit too broad. If constraints are to be added, it’s better to add them at the user level?